_posts/convars/2021-04-06-mom_player_weapon_spawn_desirability.md

```diff
@@ -9,7 +9,7 @@ default_value: 1
 ---
 
 {:.notice--info}
-This convar has no effect on weapon-focused gamemodes such as Rocket Jump, Sticky Jump, Conc, and Defrag.
+This convar has no effect on weapon-focused gamemodes such as Rocket Jump, Sticky Jump, Conc, and Defrag, and cannot be changed while the timer is running.
 
 Sets the weapons desirability for when the player spawns.
```

Comment:
Should also make note that changing the convars value will also update their carried weapons accordingly; not just set on player spawn.